html/pages/device.inc.php

```diff
@@ -170,6 +173,18 @@
             }
         }
 
+        if ($device['os'] == 'f5') {
```

Comment:
Don't need this check really, only f5 devices would have the data so the isset check below should be adequate no?